plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -323,14 +323,26 @@ class CodeWhispererCodeScanManager(val project: Project) {
         return errorMessage
     }
 
+    private fun handleCodeScanException(e: CodeWhispererCodeScanException): String? {
```

Comment:
nit: this is not handling anything, a better name might be `getCodeScanExceptionMessage`